script.js

```diff
@@ -1,35 +1,60 @@
-const books = [];
+class Books {
+  constructor(books) {
+    this.books = books;
+  }
+
+  get getBooks() {
+    return this.books;
+  }
+
+  setBooks(books) {
+    this.books = books;
+  }
+
+  remove(index) {
+    this.books.splice(index, 1);
+  }
+
+  add(book) {
+    this.books.push(book);
+  }
+}
 
 const booksEle = document.querySelector('.books');
 
 const renderBook = (book) => {
   const bookItem = document.createElement('div');
   bookItem.classList.add('book');
   const { id, title, author } = book;
+  if (id % 2 !== 0) {
+    bookItem.classList.add('gray');
+  }
   const html = `
-    <p class="title">${title}</p>
-    <p class="author">${author}</p>
+    <div class="book-info">
+      <p class="title">"${title}" </p>
+      <p class="author">by ${author}</p>
+    </div>
```

Comment:
**OPTIONAL**_ To help improve the UI/UX of your site, I suggest you make it impossible to add a book/title with an empty string, please fix this 🙏 🙏 
<img width="215" alt="cjvkvhjv" src="https://user-images.githubusercontent.com/73607512/211530684-0a5556b2-0c98-48bd-a98d-2e9cdc6b6114.PNG">

Reply:
Thanks @mckent05 . I have just fixed it.